Add nutrition facts for OIE files (#2560)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-sign-in-pwd-email/main/index.md

<StackSelector class="cleaner-selector"/>

> This guide covers the use case for a user sign-in with password and email factors, and provides a flow diagram and a sequence of integration steps.

Reviewer: ```suggestion
This guide covers the use case for a user sign-in flow with password and email factors, and provides a flow diagram and a sequence of integration steps.
```

 Nutrition Facts                                                                          |                                                                                      |
| --------------------------------------------------------------------------------  | -------------------------------------------------------------------------               |

> | Learning outcomes                     | <ul><li>Understand how to implement user sign-in with password and email factors.</li></ul>                                                       |

Reviewer: ```suggestion
| Learning outcomes                     | Understand how to implement a user sign-in flow with password and email factors.   |
```

> | What you need | <ul><li>[Okta org already configured for multifactor use case](/docs/guides/oie-embedded-common-org-setup/-/main/#set-up-your-okta-org-for-a-multifactor-use-case)</li></ul>                                                     |

Reviewer: ```suggestion
| What you need | <ul><li>[Okta org already configured for a multifactor use case](/docs/guides/oie-embedded-common-org-setup/-/main/#set-up-your-okta-org-for-a-multifactor-use-case)</li></ul>                                                     |
```
